map-prototype/index.html

```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
+  <title>Map / Philly Mesh</title>
+  <link rel="stylesheet" href="main2.css"/>
+  <link rel="stylesheet" href="style.css"/>
+  <link rel="stylesheet" href="tachyons.min.css"/>
```

Comment:
Is there a way to refactor some of this style stuff? I think the main2 has normalize and a grid system in it, and tachyons is included seperately. 

I think maybe we could gut most of this, go with one framework/toolkit and minimally define css needed on top (and keep normalize?) @garrying this is all you I got nothing else here.

Never worked with tachyons but it looks